src/Features/CSharpTest/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -935,7 +935,7 @@ class Program
             {
                 static void Main()
                 {
-                    G<int>.@class {|Rename:@class|} = new G<int>.@class();
+                    G<int>.@class {|Rename:@class|} = new();
```

Comment:
can you add sibling tests that have the same input, but which have the ImplicitObjectCreationWhenTypeIsApparent set to false, to show we get the old output in that case?